include/cantera/transport/HighPressureGasTransport.h

```diff
@@ -66,26 +258,1047 @@ class HighPressureGasTransport : public MultiTransport
      */
     void getBinaryDiffCoeffs(const size_t ld, double* const d) override;
 
-    void getMultiDiffCoeffs(const size_t ld, double* const d) override;
+    friend class TransportFactory;
+
+protected:
+
+    /**
+     * Obtain required parameters from the 'critical-parameters' species input section,
+     * and checks the critical-properties.yaml file if an acentric factor is not
+     * specified.
+     *
+     * The way that GasTransport parses the input file is that if an acentric
+     * factor is not specified, it is quietly set to 0.0. A user may have the proper
+     * acentric factor in the critical-properties.yaml file, so that file is checked if
+     * a zero value is present.
+     *
+     */
+    void getTransportData() override;
+
+    /**
+     * Computes and stores the estimate of the critical properties for each species.
+     *
+     * This method sets the species composition vector to unity for species i and zero
+     * for all other species, and then queries the thermo object for the critical
+     * properties and stores them. It then resets the composition vector to the original
+     * state. This method only needs to be called once, as the critical properties for
+     * the pure species do not change.
+     *
+     * All species must have critical properties defined in the input file, either via
+     * critical properties or by specific values of the equation of state that are
+     * not zero.
+     *
+     */
```

Comment:
Here and in several other cases, you should omit the blank line at the end:
```suggestion
     * not zero.
     */
```